### PR TITLE
add scripts for generic homegrown migration guide

### PR DESCRIPTION
**Pull request by sixhobbits**

### CODE REVIEW

File: generic/fusionAuthDockerFiles/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    networks:
+      - db_net
+    volumes:
+      - db_data:/var/lib/postgresql/data
+
+  fusionauth:
+    image: fusionauth/fusionauth-app:1.48.3
```
Comment (ghost) [inline generic/fusionAuthDockerFiles/docker-compose.yml]:
```suggestion
    image: fusionauth/fusionauth-app:latest
```

File: generic/fusionAuthDockerFiles/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+version: '3'
```
Comment (ghost) [inline generic/fusionAuthDockerFiles/docker-compose.yml]: No version needed
```suggestion

```

**Review by ghost — CHANGES_REQUESTED** [review]
@sixhobbits  There are a couple small changes on the Docker image but this paired well with the guide 😄

**Comment by ghost** [comment]
@sixhobbits conflicts please 🙏

**Comment by sixhobbits** [comment]
@alex-fusionauth fixed

**Pull request merged**